src/databricks/azext_databricks/_params.py

```diff
@@ -49,3 +50,18 @@ def load_arguments(self, _):
 
     with self.argument_context('databricks workspace wait') as c:
         c.argument('workspace_name', options_list=['--name', '-n'], id_part='name', help='The name of the workspace.')
+
+    for scope in ['create', 'show', 'delete', 'update', 'wait']:
+        with self.argument_context('databricks workspace vnet-peering {}'.format(scope)) as c:
+            c.argument('workspace_name', help='The name of the workspace.')
+            c.argument('peering_name', options_list=['--name', '-n'], help='The name of the vnet peering.')
+
+    with self.argument_context('databricks workspace vnet-peering create') as c:
+        c.argument('remote_virtual_network', options_list=['--remote-vnet'], help='The Id of the remote virtual network.')

```

Comment:
Can id or name be supported for this parameter?

Reply:
@fengzhou-msft This parameter only accepts ID now, I saw the parameter `custom_virtual_network_id` you added previously, does it support both name and ID? It seems this parameter will be passed into SDK function directly, no extra process found.

Reply:
I added the support in a [validator](https://github.com/Azure/azure-cli-extensions/blob/7c7c086dfe6da58b0e87ce9eb0eb66deb056d8b4/src/databricks/azext_databricks/_validators.py#L33).

Reply:
@fengzhou-msft Oh, I see. Good point.